.github/workflows/benchmark.yml

```diff
@@ -25,10 +25,10 @@ jobs:
     # If we're running on a PR, use ubuntu-latest - a shared runner. We can't use the self-hosted
     # runners on arbitrary PRs, and we don't want to unleash that load on the pool anyway.     
     # If we're running on main, use the OTEL self-hosted runner pool. 
-    
-    # TODO - temporarily move main to the shared workers, until we've resolved the instance setup issue
+    #
+    # TODO - temporarily use self-hosted to get shared-worker environment going
+    runs-on: 'self-hosted'
```

Comment:
PR run appears to be against a special branch:

```
Job defined at: open-telemetry/opentelemetry-rust/.github/workflows/benchmark.yml@refs/pull/2942/merge
```

can you add `dependabot/scottgerring/benchmark-fix` to the workflow definition above?

```
on: 
  pull_request:
  push:
    branches:
      - main
      - dependabot/scottgerring/benchmark-fix
```